### PR TITLE
doc; CLI; Improved CLI documentation in the main README and within the CLI.

### PR DESCRIPTION
**Pull request by steegecs**
**Context:**
The documentation on the CLI needed improving, so I improved the documentation in the main README of the repo and also the README within `subgraphs/messari-cli/docs`. Also, there was a file with some code that was not doing anything the in `messari-cli/src/common-helpers/build` folder called `writeDeployment.ts`. This functionally is encapsulated in the ScriptGenerator class. I just went ahead and removed it here.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,6 +24,7 @@ This repo contains subgraphs defined using a set of standardized schemas. These
...
+- There is also a cli-tool called `messari-cli`. It utilized The Graph CLI and exploits the standard directory structure to template and prepare subgraph deployments for you with simple commands. You can install the CLI by moving into the `messari-cli` folder and running `npm install -g ./` to install the package then `npm install` to make sure all node modules used by the package are installed properly. This installs locally by creating a reference to this package on your machine. Therefore, if the location of your local copy of the repo changes, you will want to unintall this version and reinstall in the new location. Try running `messari b/build` to be prompted for building a subgraph. Try `messari b/build -d` to be prompted to build and then deploy subgraphs. Run `messari b/build -h` to be shown a list of optional paramters you can use to bypass prompts or add extra detail about the build/deployment you would like to execute. You can find more information about the CLI in the `messari-cli/docs/` folder.
```
Comment (dmelotik) [inline README.md:27]: imo you should move the explanation of how to install and use into messari-cli/readme.md

File: messari-cli/docs/readme.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ Check out the documentation at https://github.com/infinitered/gluegun/tree/maste
 ## Commands
```
Comment (dmelotik) [inline messari-cli/docs/readme.md:9]: rename to README.md

File: README.md (hunk 1 above)
Comment (jaimehgb) [reply README.md:27]: Agree 👍 

A short comment here and a link to a more detailed README inside messari-cli might be best

Comment (steegecs) [reply README.md]: Alright. Updated - Let me know how it is!

**Review by jaimehgb — APPROVED** [review]

**Pull request merged**